copier/main.py

```diff
@@ -133,6 +143,8 @@ class Worker:
     skip_if_exists: StrSeq = ()
     cleanup_on_error: bool = True
     force: bool = False
```

Comment:
This attribute should disappear and stop being used.

However, it can be kept in helper methods (run_auto, run_copy and run_update) if you want.

Next release will be major, so there's no problem in breaking compatibility (it's broken already anyway).

I understand that making this change can turn many tests red. Do you feel like doing it?

Reply:
I was wanting to keep this as a non-breaking change, perhaps release sooner as a 5.1.1 release. 

> I understand that making this change can turn many tests red. Do you feel like doing it?

I like breaking tests. However, would prefer to that in a new PR e.g. add new feature then remove depreciated in 2 steps. That ok?

Reply:
If you want, I can release a new v 6 alpha when merged, but not a V 5.x. Currently, the master branch has diverged too much from v 5, so I'm not getting patches for it, and next release will have to be 6. However, it's still far from stable.

Regarding the split on 2 PR, we can do it if you feel more confident, but given our current situation I'm not sure it's worth it. You can just split on various commits here. But i'll let you decide 😄

Reply:
Yes, understand that `master` is very different. Have you considered a `develop` branch for v6 so that patches could still be accepted for v5?

>I understand that making this change can turn many tests red. Do you feel like doing it?

I have made the change. Currently on Windows 29 tests are failing with:

```
result = Path(self.url)
        if self.vcs == "git":
            result = Path(clone(self.url_expanded, self.ref))
            if self.ref is None:
                checkout_latest_tag(result, self.use_prereleases)
        if not result.is_dir():
>           raise ValueError("Local template must be a directory.")
E           ValueError: Local template must be a directory.
```

Spent way too much time trying to understand this one. Pushed the changes up and going to see how it fails on CI.

Reply:
> Have you considered a `develop` branch for v6 so that patches could still be accepted for v5?

Yes, but I have very constrained resources so as to maintain 2 branches, and I have no time pressure to publish v6, so for the time being I prefer to stick to a single branch.



> I have made the change. Currently on Windows 29 tests are failing with:

It seems there's a different bug that doesn't let it get there.

Reply:
thanks for pushing the fix for the tests. to be honest, i was going to abandon this work as moved onto other projects.

Reply:
You were so close... 😄 But the last mile needed some deeper knowledge on the code base, so I couldn't ask for more. Thanks for your contribution.